$NetBSD: patch-configure,v 1.10 2022/12/03 15:53:40 adam Exp $

Cripple iODBC check, we're using UnixODBC and it was getting in the way.
Fix Perl lookup.
Fix Python lookup.
Fix Ruby support.

--- configure.orig	2022-08-09 14:57:21.000000000 +0000
+++ configure
@@ -28286,7 +28286,7 @@ fi
 
 
 
-SEARCHPATH=$ODBCPATH
+SEARCHPATH=OFF_$ODBCPATH
 NAME=iodbc
 HEADER=sql.h
 LIBNAME=iodbc
@@ -34112,7 +34112,7 @@ fi
 
 			if ( test -z "$PERL" )
 			then
-				for i in "/usr/bin" "/usr/local/bin" "/usr/pkg/bin" "/usr/local/perl/bin" "/opt/sfw/bin" "/usr/sfw/bin" "/opt/csw/bin" "/sw/bin" "/boot/common/bin" "/resources/index/bin"
+				for i in "${PREFIX}"
 				do
 					if ( test -d "$i" )
 					then
@@ -34300,52 +34300,10 @@ then
 
 		pyext=""
 
-		for pyversion in \
-				"3.20" \
-				"3.19" \
-				"3.18" \
-				"3.17" \
-				"3.16" \
-				"3.15" \
-				"3.14" \
-				"3.13" \
-				"3.12" \
-				"3.11" \
-				"3.10" \
-				"3.9" \
-				"3.8" \
-				"3.7" \
-				"3.6" \
-				"3.5" \
-				"3.4" \
-				"3.3" \
-				"3.2" \
-				"3.1" \
-				"3.0" \
-				"2.9" \
-				"2.8" \
-				"2.7" \
-				"2.6" \
-				"2.5" \
-				"2.4" \
-				"2.3" \
-				"2.2" \
-				"2.1"
+		for pyversion in "${PYVERSSUFFIX}"
 		do
 
-			for pyprefix in \
-				"$PYTHONPATH" \
-				"/usr" \
-				"/usr/local" \
-				"/usr/pkg" \
-				"/usr/local/python$pyversion" \
-				"/opt/sfw" \
-				"/usr/sfw" \
-				"/opt/csw" \
-				"/sw" \
-				"/usr/freeware" \
-				"/System/Library/Frameworks/Python.framework/Versions/Current" \
-				"/boot/common"
+			for pyprefix in "${PREFIX}"
 			do
 
 				if ( test -n "$pyprefix" )
@@ -34570,7 +34528,7 @@ echo "******************************"
 echo
 echo "***** Ruby *******************"
 
-if ( test "$ENABLE_RUBY" = "yes" )
+if ( false )
 then
 
 	HAVE_RUBY=""
